src/components/components.js

```diff
@@ -1,15 +1,19 @@
 import Chosen from './Chosen.vue'
-import Navbar from './Navbar.vue'
+import Header from './Header.vue'
```

Comment:
Please remove ".vue" becauseyou have used it in another pages without this suffix.